resources/META-INF/java-deps.xml

```diff
@@ -32,6 +32,7 @@
 
         <!-- Esy -->
 
+        <projectService serviceImplementation="com.reason.esy.EsyCompiler"/>
         <projectService serviceImplementation="com.reason.esy.EsyProcess"/>
```

Comment:
isn't esy like bucklescript ? ie: it can be used outside idea product, like webstrom because there is no module configuration ? in that case it should be registered in the main plugin.xml.

also, dune tool window might probably just registered on java-deps.xml

Reply:
Ah, still wasn't thinking about the differences between these two files. Thanks for pointing this out. Moved EsyCompiler/Process back into plugin.xml.

Also moved DuneToolWindowFactory into java-deps.xml.